core/src/mast/node_fingerprint.rs

```diff
@@ -79,6 +79,7 @@ impl MastNodeFingerprint {
                         // we include the operation index to distinguish between basic blocks that
                         // would have the same assert instructions, but in a different order
                         bytes_to_hash.extend(op_idx.to_le_bytes());
+                        let inner_value: u64 = Felt::into(*inner_value);
```

Comment:
```suggestion
                        let inner_value = u64::from(*inner_value);
```

nit: using `T::into()` explicitly like this isn't canonical